### ISSUE TITLE
Midterm Report Peer Review by qs76

### ISSUE THREAD

**Issue by kevbot77** [poojachoudhary95/ORIE-4741_Project#6]
The project aims to predict the household sales area in Moscow with feature data-set that covers almost every aspect of the household. The team has done a great job overall on data pre_processing, feature engineering and its model selection. However, there are certain parts of the process that raise confusion to me and there might be room to improve. 
First of all, for the missing values handling part. The team choose to delete features that having more than 50% of missing values and also for more than 50 features. Deleting such huge amount of data can raise bias I would suggest use Educational guessing and average imputation to handle the missing values. In addition, for the noisy data part, the team's guessing is one approach but their guessing might not be accurate. For example, taking 4965 to 1965 is relatively risky and I would suggest running a pilot trials to figure out the proper range of the data with error size, then delete those outliers. 

In addition, for the feature engineering part as well as the feature selection and modeling part,  apply the one-hot encoding transformation to all categorical features and transform them into binary variables can cause problems. As it was mentioned in the previous paragraph, certain part of the data set contains categorical data set might have ascending order relationship. In that case, transformation to nominal data set should be more suitable. Moreover, it is encouraged to attempt more trials to split the dataset into training sets and test set. The team choose the ratio as 7:3, while the typical train set to test set ratio should be even higher. 

Lastly, the future work suggest that more separate models for different size of houses will be assessed and I think it is a good start. Such approach can help identify the influential order of each factor and eventually we will try to fit a model that takes as few variables as possible
